src/Containers/RestaurantsList.js

```diff
@@ -0,0 +1,79 @@
+import React,{ Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {fetchRestaurants} from '../Actions/Actions';
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import Restaurant from "../Components/Restaurant";
+
+
+class RestaurantsList extends Component{
+
+    componentDidMount() {
+       this.props.getRestaurants();
+    }
+
+    render(){
+        const { restaurants, isFetching, lastUpdated, error } = this.props;
+        return(
+            <div>
+                <p>
+                    { lastUpdated && (
+                        <span>
+                            Last updated at {new Date(lastUpdated).toLocaleTimeString()}.{' '}
```

Comment:
why do you need to `new Date`?

Reply:
doesn't work without creating a date object from the timestamp 'lastUpdate'